pyproject.toml

```diff
@@ -1,3 +1,39 @@
+[project]
+name = "nuketesting"
+version = "0.1.0"
+description = "Testing library for both running and testing plugins and gizmo's in Nuke"
+authors = [
+    { name = "Gilles Vink", email = "gilles@vinkvfx.com" }
+]
+dependencies = [
+    "click>=8.1.7",
+    "pytest>=7.4.4",
+]
+readme = "README.md"
+requires-python = ">= 3.7"
```

Comment:
Can we somehow specify the specific python versions? Like 3.7, 3.9 ? The others will not be compatible with the nuke package.

Reply:
Can only be a range. Does not support the OR operator unfortunately